convex/retryLogic.ts

```
@@ -0,0 +1,459 @@
+      break
+    case 'specificity':
+      suggestedPromptVariant = 'question_focused'
+      break
+    default:
+      // Cycle through variants
+      const variants = [
+        'standard',
+        'question_focused',
+        'confidence_calibrated',
+        'context_aware',
+      ]
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)